### DISCUSSION TITLE
Option to toggle between current time + duration in media-time-display

### DISCUSSION THREAD

**Discussion by dylanjha** [muxinc/media-chrome#181] [Proposals]
`media-time-display` is great, I can choose different options like:

- weather to show current time or remaining time
- weather to show the duration

I find myself wanting to have the option of toggling between the current time and the remaining time on click. Something like this:

https://user-images.githubusercontent.com/764988/157129835-f34abc83-93bc-41a3-91a7-d4a1f6e02086.mp4

As far as naming goes, this is what exists:

default (current time):

```html
<media-time-display>
```

current time + duration

```html
<media-time-display show-duration>
```

remaining only:

```html
<media-time-display remaining>
```

remaining + duration

```html
<media-time-display remaining show-duration>
```

I propose another option in the form of a boolean attribute:

* `toggleable`

A developer can add `toggleable` onto any of the above configurations and when clicked the time display would toggle the current time to remaining time and vice versa.

Thoughts? Or better name ideas? I could also get behind a more explicit name `toggle-remaining` (although this attribute doesn't care weather you're starting in current time display or remaining time display, it will simply swap out the alternative on click)

**Comment by luwes** [comment]
I like it, not even sure it needs a special attribute. Wouldn't it be okay it does this by default?

**Comment by heff** [comment]
I like it too, and also think this should just be the default. The negative sign makes it pretty clear what's going on. I wouldn't worry about an additional attr unless we find a use case where someone really doesn't want it to change on click. 

This is for a separate change, but some TV players show the time of day the video will end for the duration. That could be a nice toggleable feature too.

**Comment by dylanjha** [comment]
Great! Thanks @heff and @luwes  -- I'll work on this improvement!
